model/encoder_decoder.py

```diff
@@ -79,18 +84,18 @@ def __init__(
 
             embedding_customers = [
                 Embedding(
-                    num_embeddings = int(customer_info.num_cat[i] + 1),
-                    embedding_dim = int(customer_info.embedding_size[i]),
-                    max_norm = 1
+                    num_embeddings=int(customer_info.num_cat[i] + 1),
```

Comment:
these are all just black formatting. do you have it enabled?

Reply:
Yes! For me it reformats at every save. Maybe it didn't launch this time with vscode?